fearless/en.lproj/Localizable.strings

```diff
@@ -895,3 +895,4 @@ Remember to make a backup of your key and keep it in a safe and private place (e
 "all.done.alert.hash.stub" = "Hash";
 "all.done.alert.result.stub" = "Result";
 "all.done.alert.success.stub" = "Success";
+"contacts.scan" = "Scan QR code";
```

Comment:
seems to have been changed to short? "QR-code"?

Reply:
I just use design description